pkg/pipeline/pipeline_builder.go

```diff
@@ -354,6 +354,8 @@ func getWriter(opMetrics *operational.Metrics, params config.StageParam) (write.
 		writer, err = write.NewWriteNone()
 	case api.LokiType:
 		writer, err = write.NewWriteLoki(opMetrics, params)
+	case api.IpfixType:
+		writer, err = write.NewWriteIpfix(params)
```

Comment:
opMetrics is not used in ipfix export, because we export only standard fields + kube context right now. It might be added if use-cases arise for custom registries.